db/seeds.rb

```diff
@@ -6,7 +6,11 @@
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
 
-5.times do 
+users = User.create([{email: 'user_1@quotes.come', encrypted_password: 'user_1'},
+                    {email: 'user_2@quotes.come', encrypted_password: 'user_2'},
+                    {email: 'user_3@quotes.come', encrypted_password: 'user_3'}])
```

Comment:
```suggestion
                     {email: 'user_2@quotes.come', encrypted_password: 'user_2'},
                     {email: 'user_3@quotes.come', encrypted_password: 'user_3'}])
```

Reply:
I think Devise suggests to do it this way:

`{email: 'user_3@quotes.come', password: 'user_3', password_confirmation: 'user_3'}`

encryption is handled by Devise, you should not put a values there